### ISSUE TITLE
send netdata health monitoring variables to backends

### ISSUE THREAD

**Issue by n0guest** [firehol/netdata#4035]
Hello.

Based on the content of [health.d/netfilter.conf](/firehol/netdata/blob/master/conf.d/health.d/netfilter.conf#L22), I can assume, that Netdata reads net.nf_conntrack_max. And here is what is exported:

```
# curl -s 'localhost:19999/api/v1/allmetrics?format=prometheus_all_hosts' | grep netdata_netfilter | cut -f 1 -d '{' | sort | uniq
netdata_netfilter_conntrack_sockets_active_connections_average
netdata_netfilter_netlink_changes_changes_persec_average
netdata_netfilter_netlink_errors_events_persec_average
netdata_netfilter_netlink_expect_expectations_persec_average
netdata_netfilter_netlink_new_connections_persec_average
netdata_netfilter_netlink_search_searches_persec_average
```
In the same time:

```
# curl -s 'localhost:19999/netdata.conf' | grep -A2 nf_conntrack_max
[plugin:proc:/proc/sys/net/netfilter/nf_conntrack_max]
        # filename to monitor = /proc/sys/net/netfilter/nf_conntrack_max
        # read every seconds = 10
```
So, why value of nf_conntrack_max isn't exported as metric? Am I missing something?

**Comment by ktsaou** [comment]
Hi, this is a variable in netdata, so it is not sent to backends.
You are right though. It should be exported.
There is no reason to keep a full chart for it, so we need to find another way.

**Comment by ktsaou** [comment]
I implemented this in PR #4200.
Can you test it?

**Comment by ktsaou** [comment]
Updated the wiki too: https://github.com/firehol/netdata/wiki/Using-Netdata-with-Prometheus#netdata-host-variables

**Comment by n0guest** [comment]
> I implemented this in PR #4200. Can you test it?

Yep, it works. With and without "variables=yes":

```
$ diff -y /tmp/m1 /tmp/m2 | grep '>'
							      >	netdata_active_processors
							      >	netdata_ipc_semaphores_arrays_max
							      >	netdata_ipc_semaphores_max
							      >	netdata_net_eth0_nic_speed_max
							      >	netdata_netfilter_conntrack_max
							      >	netdata_tcp_max_connections
							      >	netdata_tcp_max_orphans
							      >	netdata_tcp_mem_high
							      >	netdata_tcp_mem_low
							      >	netdata_tcp_mem_pressure
```

Thank you.

PS: Sorry for delay.
